server-settings/nginx.conf

```diff
@@ -0,0 +1,41 @@
+# nginx.conf
+upstream django {
+    # connect to this socket
+    server unix:///tmp/uwsgi.sock;    # for a file socket
+    }
+
+server {
+    # the port your site will be served on
+    listen      80;
+    # the domain name it will serve for
+    server_name smartcampus.csie.ncku.edu.tw    # substitute your machine's IP address or FQDN
+    charset     utf-8;
+
+    #Max upload size
+    client_max_body_size 75M;   # adjust to taste
+
+    # Django media
+    location /media  {
+                alias /home/rapirent/smart_campus/smart_campus/smart_campus/media;      # your Django project's media files
```

Comment:
Is it possible to use `$HOME` instead of `/home/rapirent`?

Reply:
Not working

Reply:
I knew.
Hmmm...
I'll search whether there exists a better way.

Reply:
I think "in the production" we should change the MEDIA_URL from BASE_DIR to `/var/` like `/var/www/example.com/media/`, and add an line ` MEDIA_URL = "/var/www/example.com/media/"` to the setting/production.py

ref: https://docs.djangoproject.com/en/1.11/ref/settings/

Reply:
Great Job